### PR TITLE
slower mode

### PR DESCRIPTION
**Pull request by hamalhardik**
Robots and Rotini ;)))

### CODE REVIEW

**Review by Nuffa1 — APPROVED** [review]
Good work.

File: src/main/java/com/team2502/robot2023/commands/DriveCommand.java [hunk 1]
```diff
@@ -46,6 +47,8 @@ private enum Drivetype {
     private boolean slowmode =  false;
     private boolean prevTog = false;
 
+    private boolean slower = false;
+
```
Comment (hamalhardik) [inline src/main/java/com/team2502/robot2023/commands/DriveCommand.java]: Dont. make an extra line >:(

**Review by Sankalp-A — CHANGES_REQUESTED** [review]
Good code!
No need for extra line 51.

**Review by hamalhardik — COMMENTED** [review]
review guys ;)

**Comment by Sankalp-A** [comment]
> Good work.

boson

**Review by Nuffa1 — APPROVED** [review]

**Review by Sankalp-A — APPROVED** [review]
yes

**Pull request merged**